src/react/databrowser/buckets/details/Overview.tsx

```diff
@@ -162,6 +187,14 @@ function Overview({ bucket, ingestionStates }: Props) {
             : null
         }
       />
+      <VeeamCapacityModal
+        isOpen={isCapacityModalOpen}
+        close={() => {
+          setIsCapacityModalOpen(false);
+        }}
+        bucketName={bucket.name}
+        maxCapacity={prettyBytesClusterCapacity}
+      />
```

Comment:
Usually I tend to mutualize the button with the modal, so that the state managing the modal is localised to the button component

Reply:
Fixed in 49e7d8dae934e9182e815b5ad0f47c51712c374b